plugins/jira-dashboard-backend/src/api.ts

```diff
@@ -51,23 +51,27 @@ export const getFilterById = async (
 };
 
 export const getIssuesByFilter = async (
-  project: JiraProject,
+  projects: JiraProject[],
   components: string[],
   query: string,
 ): Promise<Issue[]> => {
-  const { projectKey, instance } = project;
-  const jql = jqlQueryBuilder({ project: projectKey, components, query });
-  const response = await callApi(
-    instance,
-    `${instance.baseUrl}search?jql=${jql}`,
-    {
-      method: 'GET',
-      headers: {
-        Accept: 'application/json',
+  const issues: Issue[] = [];
+  for (const project of projects) {
+    const { projectKey, instance } = project;
+    const jql = jqlQueryBuilder({ project: [projectKey], components, query });
+    const response = await callApi(
+      instance,
+      `${instance.baseUrl}search?jql=${jql}`,
+      {
+        method: 'GET',
+        headers: {
+          Accept: 'application/json',
+        },
       },
-    },
-  ).then(resp => resp.json());
-  return response.issues;
+    ).then(resp => resp.json());
+    issues.push(...response.issues);
```

Comment:
This line is was causing the error using an invalid jira.com/incoming-issues-status annotation. This because the response is undefined and therefore not iterable.

Rather than throwing an error, we want to return an empty list of issues to frontend (this is the current default behaviour on main). 

Please update this function to handle the case when response is undefined :)